### ISSUE TITLE
Conda-forge contains old version

### ISSUE THREAD

**Issue by steffen-fissler** [mims-harvard/TDC#213]
Dear developers,
thanks for all the work you put into TDC!

**Describe the bug**
When installing PyTDC via pip, one gets 0.4.1. Via conda-forge one gets 0.3.8:
With 0.3.8 in a Python console in terminal
`from tdc import Oracle`
`oracle = Oracle(name = 'QED')`
leads to
`ImportError: cannot import name 'rmsd' from 'tdc.chem_utils.oracle.oracle' (/Users/<my_user>/opt/miniconda3/envs/<my_environment>/lib/python3.11/site-packages/tdc/chem_utils/oracle/oracle.py)`

This is not the case with version 0.4.1, but that is not available on conda-forge.

Furthermore, is there any changelog available? I could not find one and the deployments do not seem to provide details about releases.

**To Reproduce**
1. Open a terminal
3. Enter `python` in an environment which has PyTDC installed via conda-forge
4. Enter `from tdc import Oracle`
5. Enter `oracle = Oracle(name = 'QED')`
6. See error `ImportError: cannot import name 'rmsd' from 'tdc.chem_utils.oracle.oracle' (/Users/<my_user>/opt/miniconda3/envs/<my_environment>/lib/python3.11/site-packages/tdc/chem_utils/oracle/oracle.py)`

**Expected behavior**
Basically having version 0.4.1 installed when using conda-forge. In the procedure we propose, a conda channel is necessary and we cannot use pip unfortunately.

**Screenshots**
Not applicable

**Environment:**
- OS: MacOS
- Python version: 3.11
- TDC version: 0.3.8
- Any other relevant information: No

**Additional context**
Thanks again for your contributions! I am grateful.

Best regards
Steffen

**Comment by amva13** [comment]
Hi Steffen, the conda installation of TDC is deprecated.
